exporters/etw/include/opentelemetry/exporters/etw/etw_provider_exporter.h

```diff
@@ -0,0 +1,644 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
nit: naming. Since it's in the `exporters` directory, it implies all code under is related to exporter.. So should we name it `etw_provider_exporter.h` or simply `etw_provider.h` ? I think a shorter name would be nicer.

Reply:
I added the `exporter` suffix because it was there for other exporter files. Can change it to `etw_provider.h`

Reply:
No, that's ok. We can change later.